src/animationBuilder.tsx

```diff
@@ -25,12 +26,52 @@ const mockTargetValues: LayoutAnimationsValues = {
   currentBorderRadius: 0,
 };
 
+function maybeReportOverwrittenProperties(
+  layoutAnimationStyle: StyleProps,
+  style: NestedArray<StyleProps>,
+  displayName: string
```

Comment:
Maybe we should rename it to `componentName`?

Reply:
We use this string from `createAnimatedComponent`:
```tsx
  AnimatedComponent.displayName = `AnimatedComponent(${
    Component.displayName || Component.name || 'Component'
  })`;
```
So I would keep the current name

Reply:
Okay, thanks for the response